### PR TITLE
[SM-4895] update docs for post sprint 42 release

### PR DESCRIPTION
**Pull request by AgnesAnn**

### CODE REVIEW

File: docs/api-documentation/V1.17/index.md [hunk 1]
```diff
@@ -0,0 +1,3069 @@
+---
+layout: default
+title: API specification V1.16
```
Comment (peadarkelly) [inline docs/api-documentation/V1.17/index.md]: This needs to be bumped to 1.17

File: docs/api-documentation/V1.17/index.md [hunk 2]
```diff
@@ -0,0 +1,3069 @@
...
+---
+# API specification
+{: .govuk-heading-xl}
+
+Version 1.16
```
Comment (peadarkelly) [inline docs/api-documentation/V1.17/index.md]: and here

File: docs/api-documentation/V1.17/index.md (hunk 1 above)
Comment (AgnesAnn) [reply docs/api-documentation/V1.17/index.md]: done

File: docs/api-documentation/V1.17/index.md (hunk 2 above)
Comment (AgnesAnn) [reply docs/api-documentation/V1.17/index.md]: done, thanks

**Pull request merged**